apps/src/templates/lessonOverview/activities/Activity.jsx

```diff
@@ -21,10 +21,16 @@ export default class Activity extends Component {
     return (
       <div>
         <h2 style={styles.activityHeader} id={`activity-${activity.key}`}>
-          {i18n.activityHeader({
-            activityName: activity.displayName,
-            activityDuration: activity.duration
+          {i18n.activityHeaderName({
+            activityName: activity.displayName
           })}
+          {activity.duration > 0 && (
```

Comment:
... and this would become `activity.duration && ...`. It might seem a little tricky because it takes advantage of the fact that `0` is falsy in JS, but you could clarify with a comment if you wanted.